anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -0,0 +1,108 @@
+= Using Cross-Region Failover
+
+Anypoint MQ Connector for Mule 4 version 4.0.5 and later provides failover capability for standard queues.
+
+This feature is not enabled by default.
+Before using this feature, review xref:mq::mq-failover.adoc[].
+
+[IMPORTANT]
+When you enable failover for a queue and the primary region goes down,
+your data resides in a different region.
```

Comment:
All the updates we made to 3.x docs need to be updated here as well..

Reply:
Yeah, I missed that one. Added now.